custom_components/rrd/utils.py

```diff
@@ -21,3 +22,25 @@ def rrd_scaled_duration(duration):
         raise vol.Invalid("Duration must be a number.")
 
     return duration
+
+
+# Source: https://stackoverflow.com/a/54331471
+UNITS = {"s":"seconds", "m":"minutes", "h":"hours", "d":"days", "w":"weeks"}
```

Comment:
these should go into `consts.py`

Reply:
Fixed in https://github.com/dgomes/rrd_recorder/pull/3/commits/540f6a7b41e15f63cd94525b5dc00ebfcfa97fc7